_includes/features.html

```diff
@@ -23,6 +23,81 @@ <h2><span class="special-text">Multi-cluster management</span> for Kubernetes an
     </div>
 </section>
 
+<section>
+    <div class="container flex">
+        <div class="text ">
+
+            <h2>Manage your connections with <span class="special-text">Environments</span></h2>
+            <p> Environments make it easier for you to manage, share, and work with a collection of resources as a group, instead of dealing with all your Connections and Credentials on an individual basis.</p>
+            <div class="button-para">
+                <a href="https://docs.meshery.io/concepts/logical/environments" class="highlight">View Docs</a>
```

Comment:
@ritiksaxena124 can you make these docs to open in another tab not in the same, it just makes user frustrated going again to site back and spoil users existing view